src/nvim/screen.c

```diff
@@ -1925,10 +1925,10 @@ static int line_putchar(LineState *s, schar_T *dest, int maxcells, bool rl)
 /// @return number of chars added to \param p
 static size_t
 fill_foldcolumn(
-    char_u *p,
-    win_T *wp,
-    foldinfo_T foldinfo,
-    linenr_T lnum
+                char_u *p,
```

Comment:
this is better than before, but ideally should be converted to nvim-style parameter lists instead (but not a show-stopper for the first batch, as it doesn't make it _worse_ at least :)

Reply:
fixed